runtime/autoload/man.vim

```diff
@@ -57,30 +53,75 @@ function! man#open_page(count, count1, mods, ...) abort
     noautocmd execute 'silent' a:mods 'split' fnameescape(bufname)
   endif
   let b:man_sect = sect
-  call s:read_page(path)
+  call s:put_page(page)
 endfunction
 
 function! man#read_page(ref) abort
   try
     let [sect, name] = man#extract_sect_and_name_ref(a:ref)
     let [b:man_sect, name, path] = s:verify_exists(sect, name)
+    let page = s:get_page(path)
   catch
     " call to s:error() is unnecessary
     return
   endtry
-  call s:read_page(path)
+  call s:put_page(page)
 endfunction
 
-function! s:read_page(path) abort
+" Handler for s:system() function.
+function! s:system_handler(jobid, data, event) abort
```

Comment:
This should be `function! s:system_handler(jobid, data, event) dict abort`.  The `dict` is required as of the partials patch merge.